deploy/internal/cloud_creds_gcp_cr.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: cloudcredential.openshift.io/v1
```

Comment:
Is there a particular reason to use these values of names and namespaces instead of following the convention of other CR creations (Azure and AWS):
```
openshift-ingress-gcp,
openshift-cloud-credential-operator,
cloud-credentials, 
openshift-ingress-operator
```

Reply:
done, I have changed the naming conventions according to the Azure and AWS